src/launcher/launcher_main.cpp

```diff
@@ -445,7 +448,7 @@ void mouse_click() {
 		return;
 	case ui_obj_play_game:
 		if(file_is_ready.load(std::memory_order::memory_order_acquire) && !selected_scenario_file.empty()) {
-			if(IsProcessorFeaturePresent(PF_AVX512F_INSTRUCTIONS_AVAILABLE)) {
```

Comment:
This shouldn't be necessary, since if the CPU doesn't support avx, it certainly doesn't support avx512, and so this should return false.